### ISSUE TITLE
About the fusion of depth net and posenet

### ISSUE THREAD

**Issue by TensorFlowWangHT** [Ougui9/DFineNet#1]
Hello, I am very interested in your work, because I have done similar research before, also using F.ma code. However, I have always had problems when blending posenet and depth net, that is, the part without the lidar will calculate the wrong depth information through the photometric error, and the pseudo-color display is always dark blue. Would you like to ask if you have encountered the same problem in the process of research?

**Comment by Ougui9** [comment]
Hi,
Thank you for your interest in our work! I did not have this problem in my experiments, except that the boundary part of depth estimation could be far off GT in most cases since we merely have sparse depth measurement for the boundary pixels. However, the far-off-GT cases should not happen to the pixels surrounded by depth measurement because there is no constraint forcing the model to be trained in that way.

 So I will suggest to check the loss function, and see if you probably forgot to zero out the loss from pixels without depth measurement.

**Comment by TensorFlowWangHT** [comment]
Thank you very much for your reply. I feel that I still need to learn a lot. I am waiting for you to publish the code to compare my own code.
